src/features/services/store.ts

```diff
@@ -111,3 +114,32 @@ export const useServicesStore = create<ServicesState & ServicesActions>(
       })),
   })
 );
+
+interface DataServiceState {
+  keys: string[];
+}
+
+interface DataServiceActions {
+  addKey: (key: string) => void;
+  delKey: (key: string) => void;
+}
+
+export const useDataServiceStore = create<
+  DataServiceState & DataServiceActions
+>(
+  persist(
+    (set) => ({
+      keys: [],
+      addKey: (key: string) => set((s) => ({ keys: [...s.keys, key] })),
+      delKey: (key: string) =>
+        set((s) => ({ keys: s.keys.filter((k) => k !== key) })),
+    }),
+    {
+      name: "GWEN.SERVICES.DATA",
```

Comment:
You can look in the browser storage under this key to confirm keys are being written.